test/model/test/common/util.cpp

```diff
@@ -180,14 +180,18 @@ verify_workload(const model::kv_workload &workload, TEST_OPTS *opts, const std::
 {
     /* Run the workload in the model. */
     model::kv_database database;
-    workload.run(database);
+    std::vector<int> ret_model = workload.run(database);
 
     /* When we load the workload from WiredTiger, that would be after running recovery. */
     database.restart();
 
     /* Run the workload in WiredTiger. */
     testutil_recreate_dir(home.c_str());
-    workload.run_in_wiredtiger(home.c_str(), env_config);
+    std::vector<int> ret_wt = workload.run_in_wiredtiger(home.c_str(), env_config);
+
+    /* Compare the return codes. */
+    if (ret_model != ret_wt)
+        throw model::model_exception("The return codes differ.");
```

Comment:
Please put the return codes in the exception text.

Reply:
Oh, this was actually supposed to be done via `testutil_assert`. This is just a quick utility function for the model's own unit tests (as opposed to something that is actually intended to find errors in WT), so aborting on error and dumping the core is more in line what we should do here.

Reply:
I made the change in [4ff2d8b](https://github.com/wiredtiger/wiredtiger/pull/10329/commits/4ff2d8b72c03854624bc022dd63268a71afcffbc).